### DISCUSSION TITLE
GitHub Actions is currently disabled

### DISCUSSION THREAD

**Discussion by Zeno-sole** [community/community#53591] [Actions]
### Select Topic Area

Question

### Body

deepin-community  GitHub Actions is currently disabled
We don't know why we were banned, and no one responded to the ticket

**Comment by Zeno-sole** [comment]
https://github.com/deepin-community  
We have not abused Actions subjectively. If there are problems with some workflows, we can modify them

**Reply by 8MiYile** [reply]
All the Repo's warn that

GitHub Actions is currently disabled for this repository. Please reach out to [GitHub Support](https://support.github.com/contact?tags=dotcom-actions) for assistance.

**Reply by mspist** [reply]
I have the same issue!

**Comment by Zeno-sole** [comment]
Reply from the official：

Hello,

Thank you for writing in.

The account has restrictions imposed because it appears to have been taking part in activity which goes against our Terms of Service:

[Terms of Service](https://docs.github.com/github/site-policy/github-terms-of-service).

In particular, any repositories that use GitHub Actions solely to interact with 3rd party websites, to engage in incentivized activities, or for general computing purposes may fall afoul of the GitHub Additional Product Terms:

[GitHub Additional Product Terms](https://docs.github.com/site-policy/github-terms/github-terms-for-additional-products-and-features#actions)

For this reason, I'm afraid we will not be removing the restrictions from this account.

Best regards,
GitHub Support

**Comment by Zeno-sole** [comment]
has been lifted
